android/java/org/chromium/chrome/browser/playlist/BUILD.gn

```diff
@@ -0,0 +1,170 @@
+# Copyright (c) 2024 The Brave Authors. All rights reserved.
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import("//build/config/android/rules.gni")
+
+android_resources("playlist_resources") {
+  sources = [
+    "res/drawable/above_shadow.xml",
+    "res/drawable/below_shadow.xml",
+    "res/drawable/default_dot.xml", #note: possible dupe
```

Comment:
@deeppandya the ones marked `note: possible dupe` I saw in the `brave/android/brave_java_resources.gni` twice. I'm not sure if it's a duplicate or not... so left it in. For those ones, instead of moving the file under `res/` I did a copy.